styles/style.css

```diff
@@ -1,15 +1,528 @@
+  text-decoration: underline red;
+  color: black;
+}
+
+.show-case-article p a {
+  color: #f72210;
+  text-decoration: none;
+}
+
+.footer-bottom ul li a {
+  color: rgb(107, 107, 107);
+  padding: 0 10px;
+  border-right: rgb(206, 206, 206) 1px solid;
+}
+
+.editor-picks-article figure div p a {
+  color: #f72210;
+  text-decoration: none;
+}
+
+.different-articles-title a:hover {
+  color: #f72210;
+  text-decoration: none;
+}
+
+.v-grid div a:hover {
+  text-decoration: underline red;
+  color: black;
+}
+
+/* nav-med */
+
+#nav-md,
+#nav-sm {
+  background-color: #f72210;
+  padding: 5px 0;
+}
+
+#nav-sm {
+  padding: 3px 0;
+}
+
+.nav-md-right a > i,
+.nav-sm-right a > i {
+  color: rgba(233, 233, 233, 0.5);
+}
+
+.nav-md-right > div > *,
+.nav-sm-right > div > * {
+  margin-left: 15px;
+}
+
+.nav-sm-right button {
+  font-size: 12px;
+}
+
+div#nav-list {
+  margin: -10px 0 0;
+  z-index: 2;
+  position: relative;
+  background-color: white;
+  padding-top: 15px;
+}
+
+#nav-list ul {
+  background-color: white;
+  width: 87%;
+}
+
+#nav-list li {
+  flex-grow: 1;
+  display: flex;
+  justify-content: center;
+}
+
+.footer-top-left p {
+  font-size: 14px;
+  color: rgb(134, 134, 134);
+}
+
+.showcase-mini-grid figcaption p {
+  font-weight: 500;
+  color: gray;
+}
+
+.footer-bottom ul:first-child li:first-child a,
+.footer-bottom ul:first-child li:last-child a,
+.footer-bottom ul:last-child li:last-child a {
+  border: none;
+}
+
+#nav-list li a {
+  color: black;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 19px;
+}
+
+.search input {
+  width: 60px;
+  border: none;
+  font-size: 19px;
+}
+
+.search button {
+  background-color: white;
+  border: none;
+}
+
+#nav-sub-list {
+  background-color: white;
+  margin: 2px 0;
+  padding-left: 15px;
+}
+
+#nav-sub-list li a {
+  color: grey;
+  padding: 0 20px;
+  text-decoration: none;
+  font-weight: bold;
+}
+
+/* ************************ showcase column ******/
+
+.show-case {
+  background-color: white;
+  margin-top: 3px;
+}
+
+.show-case-title {
+  color: rgb(153, 153, 153);
+  text-transform: uppercase;
+  font-weight: 600;
+}
+
+.editor-picks-article figure div p {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.editor-picks-article figure div p,
+.show-case-article div p:first-of-type {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  padding: 5px 10px;
+  background-color: rgb(243, 243, 243);
+  color: #f72210;
+  font-weight: bolder;
+}
+
+h4 {
+  font-weight: bold !important;
+  color: black;
+  padding-top: 10px;
+}
+
+h5 {
+  font-weight: 600 !important;
+  color: black;
+}
+
+.showcase-mini-story {
+  border-top: 1px rgb(202, 202, 202) solid;
+}
+
+.debate {
+  border-top: 4px rgb(211, 211, 211) solid;
+  border-bottom: 4px rgb(211, 211, 211) solid;
+}
+
+#showcase-column3 figure img,
+#showcase-column3 figure:first-child img {
+  float: left;
+  width: 65px;
+  margin-right: 15px;
+}
+
+#showcase-column3 > .debate figure:nth-of-type(2) img {
```

Comment:
use classes instead of long CSS selectors, a CSS selector should not have more than 3 composed selectors.